test/jdk/javax/accessibility/JComboBox/TestJComboBoxScreenMagnifier.java

```diff
@@ -0,0 +1,86 @@
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+/*
+ * @test
+ * @bug 8283214
+ * @library /java/awt/regtesthelpers
+ * @build PassFailJFrame
+ * @requires (os.family == "mac")
+ * @summary Verifies if JComboBox selected item magnifies using
+ * screen magnifier a11y tool.
+ * @run main/manual TestJComboBoxScreenMagnifier
+ */
+
+import java.awt.BorderLayout;
+import java.awt.GridLayout;
+import java.lang.reflect.InvocationTargetException;
+
+import javax.swing.JComboBox;
+import javax.swing.JFrame;
+import javax.swing.JLabel;
+import javax.swing.JPanel;
+import javax.swing.SwingUtilities;
+
+public class TestJComboBoxScreenMagnifier {
+    private static JFrame frame;
+    private static final String INSTRUCTIONS =
+            "1) Enable Screen magnifier on the Mac \n\n" +
+                "System Preference -> Accessibility -> Zoom -> " +
+                "Select \"Enable Hover Text\"\n\n" +
+            "2) Move the mouse over the combobox selected item by pressing  " +
+                "\"command\" button.\n\n" +
+            "3) If magnified label is visible, Press Pass else Fail.";
+
+    public static void main(String[] args) throws InterruptedException,
+             InvocationTargetException {
+        PassFailJFrame passFailJFrame = new PassFailJFrame(
+                "JComboBox Screen Magnifier Test Instructions", INSTRUCTIONS, 5, 12, 40);
+
+        SwingUtilities.invokeAndWait(TestJComboBoxScreenMagnifier::createAndShowUI);
+        passFailJFrame.awaitAndCheck();
+    }
+
+    private static void createAndShowUI() {
+        frame = new JFrame("JComboBox A11Y Screen Magnifier Test");
+
+        String[] fruits = new String[] {"Apple", "Orange",
+                "Mango", "Pine Apple", "Banana"};
+        JComboBox<String> comboBox = new JComboBox<String>(fruits);
+        JPanel fruitPanel = new JPanel(new GridLayout(1, 2));
+        JLabel fruitLabel = new JLabel("Fruits : ", JLabel.CENTER);
+        fruitLabel.getAccessibleContext().setAccessibleName("Fruits Label");
+        fruitPanel.add(fruitLabel);
+        fruitPanel.add(comboBox);
+        comboBox.getAccessibleContext().setAccessibleName("Fruit jCombobox");
+        frame.getContentPane().add(fruitPanel, BorderLayout.CENTER);
+
+        PassFailJFrame.addTestWindow(frame);
+        PassFailJFrame.positionTestWindow(frame,
+                PassFailJFrame.Position.HORIZONTAL);
+        frame.setLocationRelativeTo(null);
```

Comment:
```suggestion
```
It's not needed, the location of the frame is set explicitly by the above call to `PassFailJFrame.positionTestWindow`.

Reply:
Updated.